### PR TITLE
Added logging and metrics upgrading missing parameters

### PR DESCRIPTION
**Pull request by bfallonf**
For: 
https://bugzilla.redhat.com/show_bug.cgi?id=1486255
https://bugzilla.redhat.com/show_bug.cgi?id=1486266

cc @thom-at-redhat 

Will verify in the BZs.

### CODE REVIEW

**Comment by bfallonf** [comment]
Verified. @openshift/team-documentation PTAL

**Comment by bfallonf** [comment]
Second commit for: https://bugzilla.redhat.com/show_bug.cgi?id=1441049

File: install_config/upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -526,15 +526,32 @@ following required variable within the `[OSEv3:vars]` section:
...
-openshift_logging_install_logging=true <1>
-openshift_logging_image_version=<tag> <2>
+openshift_hosted_logging_deploy=true <1>
+openshift_hosted_logging_deployer_prefix=registry.example.com:8888/openshift3/ <2>
+openshift_hosted_logging_deployer_version=3.6.0 <3>
 ----
 <1> Enables the ability to upgrade the logging stack.
-<2> Replace `<tag>` with `{latest-int-tag}` for the latest version.
+<2> Replace `registry.example.com:8888/openshift3/` with your desired prefix. 
+<3> Replace with the desired image version, or with `{latest-int-tag}` for the latest version.
```
Comment (mburke5678) [inline install_config/upgrading/automated_upgrades.adoc]: To be consistent with <1> and <2>, should this be `Replace 3.6.0 with the desired image version, or with `{latest-int-tag}` for the latest version`?

File: install_config/upgrading/automated_upgrades.adoc [hunk 2]
```diff
@@ -526,15 +526,32 @@ following required variable within the `[OSEv3:vars]` section:
...
 
-. Add any other `openshift_logging_*` variables that you want to specify to
-override the defaults, as described in
-xref:../../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables].
+. Add any other `openshift_logging_*` or `openshift_hosted_logging_*` variables
+that you want to specify to override the defaults, as described in
```
Comment (mburke5678) [inline install_config/upgrading/automated_upgrades.adoc]: remove "to specify"? `that you want to override the defaults` _sounds_ cleaner and the `Add any` should cover the `to specify`.

**Comment by mburke5678** [comment]
@bfallonf Couple of nits

**Comment by bfallonf** [comment]
Thanks @mburke5678 . Changed to suggestion. I'll merge.

**Comment by bfallonf** [comment]
Third commit for: https://bugzilla.redhat.com/show_bug.cgi?id=1486938

**Pull request merged**